app/views/devise/shared/_error_messages.html.slim

```diff
@@ -0,0 +1,6 @@
+- if resource.errors.any?
```

Comment:
this file should be used for `flash` messages. But errors for object should be allocated just under the appropriate fields

Reply:
It is used for displaying total numbers of errors at the sign up page. Should I remove it?